Cannot install 2.4.0

`npm install gamemaker-rubber@2.3.0` works just fine, but `npm install gamemaker-rubber@2.4.0` (or leaving off the version) results in an error that prevents install. The error looks like this:

```
npm ERR! path {project_dir}/node_modules/gamemaker-rubber/out/cli.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '{project_dir}/node_modules/gamemaker-rubber/out/cli.js'
npm ERR! enoent This is related to npm not being able to find a file.
```
Where `{project_dir}` is the absolute path to my project.

I went through a git diff of those two versions, but couldn't find anything that would explain it. I don't use typescript in my day-to-day though.

--- a/package.json
+++ b/package.json
@@ -3,6 +3,12 @@
 	"version": "2.4.0",
 	"description": "GMS2 project compilation",
 	"main": "out/rubber.js",
+	"scripts": {
+		"prepare": "./node_modules/typescript/bin/tsc"
+	},
+	"files":[
+		"/out"
+	],
 	"dependencies": {
 		"chalk": "^2.4.1",
 		"cli": "^1.0.1",

